src/curated-corpus/helpers/topics.test.ts

```diff
@@ -94,7 +94,7 @@ describe('helper functions related to topics', () => {
 
     it('returns a full list of topics if requested', () => {
       const topics = getGroupedTopicData(data);
-      expect(topics).to.be.an('array').with.lengthOf(17);
+      expect(topics).to.be.an('array').with.lengthOf(16);
```

Comment:
should I get rid of the `'returns a list of topics without "Coronavirus" if requested'` test case as we don't show this topic?

Reply:
I think we should if this topic is no longer present anywhere in the rest of the codebase now!